packages/apps/tools/src/services/accounts/get-transactions.ts

```diff
@@ -26,17 +26,22 @@ export async function getTransactions(options: {
   account: string;
 }): Promise<ITransaction[]> {
   debug(getTransactions.name);
-
   const { network, chain, account } = options;
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
This can be removed

Reply:
This still gives me this error: 
ESLint: React Hook "useWalletConnectClient" is called in function "describeModule" that is neither a React function component nor a custom React Hook function.

In order not to use this here this way, maybe I can import the `networksData` into the component and give it as a parameter to this function? Or just supress the warning as I have done? Please let me know
@ash-vd

Reply:
The error is correct, I would do as you suggested Kristina, provide it as a parameter 👍